src/avatar.scss

```diff
@@ -97,17 +97,21 @@ $block: #{$fd-namespace}-avatar;
 
   // placeholder background
   &--placeholder {
-    background-color: var(--sapContent_ImagePlaceholderBackground);
+    background-color: var(--fdAvatar_Placeholder_BackgroundColor);
+    border: var(--fdAvatar_Border);
+    color: var(--fdAvatar_Placeholder_DefaultColor);
```

Comment:
We can also remove this color - I think sorting the default background out will actually solve most of the color contrast issues.